2020-Contributor-Survey.md

```diff
@@ -0,0 +1,124 @@
+# DRAFT -- 2020 Node.js Contributor Survey
+
+* How long have you been contributing to Node.js?
```

Comment:
Can we open by defining “contribution” to make sure it doesn’t just include code?

Reply:
Absolutely.